deckhouse-controller/pkg/controller/module-controllers/source/controller.go

```diff
@@ -412,7 +411,7 @@ func (r *reconciler) deleteModuleSource(ctx context.Context, source *v1alpha1.Mo
 		if source.GetAnnotations()[v1alpha1.ModuleSourceAnnotationForceDelete] != "true" {
 			for _, module := range source.Status.AvailableModules {
 				if err := r.cleanSourceInModule(ctx, source.Name, module.Name); err != nil {
-					r.log.Errorf("failed to clean source in the '%s' module, during deleting the '%s' module source: %v", module.Name, source.Name, err)
+					r.log.Errorf("failed to clean source in the '%s' module during deleting the '%s' module source: %v", module.Name, source.Name, err)
```

Comment:
```suggestion
					r.log.Errorf("failed to clean source in the %q module during deleting the %q module source: %w", module.Name, source.Name, err)
```

Reply:
why use %q instead of %s for a string argument and wrapping when printing?

Reply:
%q is made specially for quoting the variables. From the fmt docs:
`%q	a single-quoted character literal safely escaped with Go syntax.`
So, it's the same behavior but it would be safely. 
For example, it would escape `\n` in the string variable
```
fmt.Printf("%q\n", "hello\n;")
```
will print
```
"hello\n;"
```
while the 
```
fmt.Printf("'%s'\n", "hello\n;")
```
will print
```
'hello
;'
```